vendor/github.com/open-policy-agent/cert-controller/pkg/rotator/rotator.go

```diff
@@ -218,7 +219,7 @@ func (cr *CertRotator) refreshCertIfNeeded() error {
 			return false, errors.Wrap(err, "acquiring secret to update certificates")
 		}
 		if secret.Data == nil || !cr.validCACert(secret.Data[caCertName], secret.Data[caKeyName]) {
-			crLog.Info("refreshing CA and server certs")
+			crLog.Info("MMM refreshing CA and server certs")
```

Comment:
is this a manual edit to the file?

Reply:
this was not supposed to get it, debug code for another issue
fixed